Mathlib/Algebra/Category/Ring/BinaryCoproducts.lean

```diff
@@ -0,0 +1,91 @@
+/-
+Copyright (c) 2024 Justus Springer. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Justus Springer
+-/
+import Mathlib.Algebra.Category.Ring.Basic
+import Mathlib.Algebra.Lie.OfAssociative
+import Mathlib.CategoryTheory.Limits.Shapes.BinaryProducts
+import Mathlib.RingTheory.TensorProduct.Basic
+
+/-!
+# The tensor product is a coproduct in the category of commutative rings
```

Comment:
I think the contents of this file can go in `Algebra/Category/Ring/Constructions`

Reply:
You're right, will change it in the other PR.